php/Dockerfile.j2

```diff
@@ -25,7 +25,7 @@ RUN echo "Starting ..." && \
     echo "@edge-community http://nl.alpinelinux.org/alpine/edge/community" >> /etc/apk/repositories && \
     apk add --update --upgrade alpine-sdk apk-tools@edge-main autoconf bash bzip2 cyrus-sasl-dev curl freetype-dev gettext git \
         gnu-libiconv@edge-community icu-dev@edge-main libjpeg-turbo-dev libmcrypt-dev libmemcached-dev libpng-dev libssh2-dev \
-        libxml2-dev libzip-dev make mysql-client openssh-client patch postgresql-dev rsync tzdata && \
+        libxml2-dev libzip-dev make mysql-client openssh-client patch postgresql-dev rsync tzdata libcrypto1.1 libssl1.1 && \
```

Comment:
Please sort by name

Reply:
done

Reply:
I have no idea why the CI fail...  I was build and test the php image in local successfully. :/